scripts/lib/CIME/SystemTests/system_tests_common.py

```diff
@@ -388,7 +388,8 @@ def _compare_baseline(self):
 
             # compare memory usage to baseline
             newestcpllogfiles = self._get_latest_cpl_logs()
-            memlist = self._get_mem_usage(newestcpllogfiles[0])
+            if len(newestcpllogfiles) > 0:
+                memlist = self._get_mem_usage(newestcpllogfiles[0])
```

Comment:
Could this result in a variable-unset error below for memlist?

Reply:
No - memlist is not used if the newestcpllogfiles array is empty.